docs/source/models/causes/alzheimers/dementia_si_model_gbd_2021/index.rst

```diff
@@ -0,0 +1,292 @@
+..
+  Section title decorators for this document:
+
+  ==============
+  Document Title
+  ==============
+
+  Section Level 1 (#.0)
+  +++++++++++++++++++++
+
+  Section Level 2 (#.#)
+  ---------------------
+
+  Section Level 3 (#.#.#)
+  ~~~~~~~~~~~~~~~~~~~~~~~
+
+  Section Level 4
+  ^^^^^^^^^^^^^^^
+
+  Section Level 5
+  '''''''''''''''
+
+  The depth of each section level is determined by the order in which each
+  decorator is encountered below. If you need an even deeper section level, just
+  choose a new decorator symbol from the list here:
+  https://docutils.sourceforge.io/docs/ref/rst/restructuredtext.html#sections
+  And then add it to the list of decorators above.
+
+.. _2021_cause_alzheimers_and_dementia:
+
+==================================================
+Alzheimer's disease and other dementias (GBD 2021)
+==================================================
+
+.. contents::
+  :local:
+
+.. list-table:: Abbreviations
+  :widths: 15 15 15
+  :header-rows: 1
+
+  * - Abbreviation
+    - Definition
+    - Note
+  * - AD
+    - Alzheimer's Disease
+    -
+  * - YLD
+    - Years Lived with Disability
+    -
+  * - YLL
+    - Years of Life Lost
+    -
+
+Disease Overview
+++++++++++++++++
+
+GBD 2021 Modeling Strategy
+++++++++++++++++++++++++++
+
+Restrictions
+------------
+
+The following table describes any restrictions in GBD 2021 on the
+effects of this cause (such as being only fatal or only nonfatal), as
+well as restrictions on the ages and sexes to which the cause applies.
+
+.. list-table:: GBD 2021 Cause Restrictions
+  :widths: 15 15 20
+  :header-rows: 1
+
+  * - Restriction Type
+    - Value
+    - Notes
+  * - Male only
+    - False
+    -
+  * - Female only
+    - False
+    -
+  * - YLL only
+    - False
+    -
+  * - YLD only
+    - False
+    -
+  * - YLL age group start
+    - 40 to 44
```

Comment:
I suppose we could start the sim at age 40 as well then, although it does not make a difference when we are using the with-condition-population component.